rust/src/plane.rs

```diff
@@ -180,21 +180,21 @@ impl NcPlaneOptions {
     }
 
     /// `NcPlaneOptions` simple constructor with horizontal alignment
-    pub fn new_halign(y: i32, x: i32, rows: u32, cols: u32) -> Self {
+    pub fn new_halign(y: i32, align: NcAlign, rows: u32, cols: u32) -> Self {
         Self::with_all_options(
             y,
-            x,
+            align,
             rows,
             cols,
             NCPLANE_OPTION_HORALIGNED,
         )
     }
 
     /// `NcplaneOptions` constructor
-    pub fn with_all_options(y: i32, x: i32, rows: u32, cols: u32, flags: u64) -> Self {
+    pub fn with_all_options(y: i32, align: NcAlign, rows: u32, cols: u32, flags: u64) -> Self {
```

Comment:
not in this function, revert to `x: i32`

Reply:
Done, but did you see that it wanted `x` as `u32` in `new()` above? Is that expected?

Reply:
Nevermind, got it, my dumbness.